test/utils.ts

```diff
@@ -71,27 +71,27 @@ test('getDiscussionNumber', t => {
 	]);
 	for (const [url, result] of pairs) {
 		location.href = url;
-		t.is(result, utils.getDiscussionNumber(url));
+		t.is(result, utils.getDiscussionNumber());
```

Comment:
Types detected that the url is not passed in. Internally, it reads from window.location.